src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -55,13 +54,122 @@
                               Style="{StaticResource SchemeGridStyle}">
                             <Grid.ColumnDefinitions>
                                 <ColumnDefinition Width="Auto" />
-                                <ColumnDefinition Width="*" />
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
-                            <TextBlock Grid.Column="0"
-                                       VerticalAlignment="Center"
-                                       Style="{StaticResource SettingsPageItemHeaderStyle}"
-                                       Text="{x:Bind Name, Mode=OneWay}" />
+                            <Border Height="48"

```

Comment:
Sorta a silly question, but... can you achieve the same effect by just putting a `Padding`, `Background` and `CornerRadius` on the `Grid` itself? It would reduce the code churn a bit, and remove a redundant UI element.

Reply:
Oh yeah that works! Though for some reason I had to set the top padding to 11 and the bottom padding to 8 to get it to center properly (even though VerticalAlignment is already set to Center...)